Lib/importlib/_bootstrap_external.py

```diff
@@ -1267,7 +1267,7 @@ def set_data(self, path, data, *, _mode=0o666):
                 return
         try:
             _write_atomic(path, data, _mode)
-        except (OSError, ValueError) as exc:
+        except OSError as exc:
```

Comment:
Is it right? The documentation says "do not propagate ValueError", and ValueError can be raised in `_write_atomic()` if the last component of the path contains NUL or too long.

Reply:
Nope, it's not right! (I'm not right). I should have kept this one (now I remember: I first added the ValueError guard here but then was "should have also done in _write_atomic", but this second thinking is wrong).